README.md

```diff
@@ -5,4 +5,5 @@
 ## Implementations
 
 - [dif](https://github.com/decentralized-identity/lds-ecdsa-secp256k1-2019.js)
-- [digitalbazaar](https://github.com/digitalbazaar/secp256k1-key-pair)
\ No newline at end of file
+- [digitalbazaar](https://github.com/digitalbazaar/secp256k1-key-pair)
+- [spruce](https://docs.rs/ssi/0.2.0/ssi/ldp/struct.EcdsaSecp256k1Signature2019.html)
```

Comment:
Pardon my unfamiliarity with docs.rs and rust, but is that the best link for an implementation?  Looks like just API docs to me.  Maybe also link to https://github.com/spruceid/ssi or wherever the code lives?

Reply:
I'm not sure... Originally I was going to use this link: https://github.com/spruceid/ssi/blob/15e944620e20b31b4644edad094e01ff7b418e44/src/ldp.rs#L822 but then thought it might be nicer to use the docs page. In a future version these docs should have more human-readable content. The docs.rs pages have links to the source/repository in the navigation. But maybe it would be better to just to link to the repo, as that is more intended to be an entry point...

Reply:
Heh, that repo link was not obvious to me.  I used a search engine.  It also seems a specific 0.2.0 link would get stale easily as the lib matures.  And this is getting off topic, but implementation links should probably list the implementation language(s).

Reply:
Good point about the version. A wildcard could be used in the docs.rs URL to refer to the latest version, but the latest release of this crate doesn't have the docs building currently. Updated to use the repo link: bc3722287bf3eef9226c708a690a873a1fefd2ac.

About listing the implementation languages, sure why not: 638538a58dad33310bbb3c8ceaec0c067b50f182